eng/pipelines/extra-platforms/runtime-extra-platforms-ioslikesimulator.yml

```diff
@@ -25,9 +25,7 @@ jobs:
     platforms:
     - iossimulator_x64
     - tvossimulator_x64
-    # don't run tests on arm64 PRs until we can get significantly more devices
-    - ${{ if eq(variables['isRollingBuild'], true) }}:
-      - iossimulator_arm64
+    - iossimulator_arm64
```

Comment:
Just for clarification: Are we sure we now have the right capacity?

Reply:
It wasn't timing out on this PR, so I thought we can try it on main and see how it goes, we can always disable again in the future.

Reply:
I like the approach. I can't remember how many arm64 machines we had back when it was disabled, but it's a safe bet we had way less.